price/interfaces/graphql/schema.graphql

```diff
@@ -13,4 +13,17 @@ type Commerce_Charge {
 type Commerce_ChargeQualifier {
     type: String!
     reference: String!
-}
\ No newline at end of file
+}
+
+input Commerce_ChargeQualifierInput {
+    type: String!
+    reference: String!
+}
+
+type Commerce_Price_Charges {
+    items: [Commerce_Charge!]
+    hasType(ctype: String): Boolean
+    hasChargeQualifier(qualifier: Commerce_ChargeQualifierInput!): Boolean
+    getByChargeQualifierForced(qualifier: Commerce_ChargeQualifierInput!): Commerce_Charge
```

Comment:
Why can we not reuse the type Commerce_ChargeQualifier ? Would make sense to me - it is meant as a "key" that qualifies one Charge in a list of Charges

Reply:
In graphql input types follow an naming convention like in this case you have to suffix your input type with "...Input"

Therefore I've tried to prevent a breaking change because the Commerce_ChargeQualifier type was already introduced but never used.